tests/integration/Makefile

```diff
@@ -0,0 +1,17 @@
+modules   := $(wildcard molecule/module_*)
+roles     := $(wildcard molecule/role_*)
+scenarios := $(modules) $(roles)
+
+
+.PHONY: all
+all: $(scenarios)
+
+.PHONY: modules
+modules: $(modules)
+
+.PHONY: roles
+roles: $(roles)
+
+.PHONY: $(modules) $(roles)
```

Comment:
I wasn't aware this is possible (and now that I'm looking at it - that it's actually pretty simple to achieve) - awesome! I always found myself temporarily hard-coding the module name somewhere...